python/sglang/srt/model_executor/cuda_graph_runner.py

```diff
@@ -550,7 +551,8 @@ def replay_prepare(
         self.out_cache_loc[:raw_num_token].copy_(forward_batch.out_cache_loc)
         self.positions[:raw_num_token].copy_(forward_batch.positions)
         num_token_non_padded = len(forward_batch.input_ids)
-        self.num_token_non_padded[...] = num_token_non_padded
+        if enable_num_token_non_padded(self.model_runner.server_args):
+            self.num_token_non_padded[...] = num_token_non_padded
```

Comment:
copy from `forward_batch.num_token_non_padded` is better because it is GPU-GPU copy,
this statement incurs some GPU-CPU copy